libs/remix-lib/src/types/ICompilerApi.ts

```diff
@@ -34,7 +34,7 @@ export interface ICompilerApi {
 
     resolveContentAndSave: (url: string) => Promise<string>
     fileExists: (file: string) => Promise<boolean>
-    writeFile: (file: string, content: string) => Promise<void>
+    writeFile: (file: string, content: any) => Promise<void>
```

Comment:
is this change necessary?

Reply:
Yes, when left as a string, It becomes difficult to write the pdf to file explorer.  Even though deep in the recesses of the plugin engin, it takes a string. Making it any allows me to write the result from generating the pdf to fileexplorer.